RandomUsersTest/app/src/main/java/com/skaravatskiy/randomusers/viewmodel/UserListViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.skaravatskiy.randomusers.viewmodel
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import com.skaravatskiy.randomusers.data.model.database.User
+import com.skaravatskiy.randomusers.data.repository.Repository
+import com.skaravatskiy.randomusers.data.repository.RepositoryDefault
+import com.skaravatskiy.randomusers.ui.userslist.Sort
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class UserListViewModel : RxViewModel(), BaseViewModel {
+    var repository: Repository = RepositoryDefault()
+    private var userData: MutableLiveData<List<User>>? = null
+    lateinit var errorResponse: Throwable
+    override fun getData(): LiveData<List<User>> {
```

Comment:
It would be great to have more more specific name of the method because by this name wi don't know what kind of data will we retrieve.Please use smth like `getUser()`